packages/vue/src/components/atoms/SfRating/SfRating.html

```diff
@@ -1,12 +1,12 @@
 <div class="sf-rating">
   <!--@slot custom icon for finalScore. Provide single icon that will be automatically repeated -->
   <slot name="icon-positive" v-if="finalScore && finalMax" v-for="index in parseInt(finalScore, 10)">
-    <svg width="14" height="14" viewBox="0 0 11 10" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M5.5 0L6.73483 3.80041H10.7308L7.49799 6.14919L8.73282 9.94959L5.5 7.60081L2.26718 9.94959L3.50201 6.14919L0.269189 3.80041H4.26517L5.5 0Z" class="sf-rating__icon-positive"/>
+    <SfIcon size="14px" icon="star" color="green-primary"></SfIcon>
```

Comment:
What if someone wants to change the color of active icons? color should be set to default accent imho

Reply:
It is possible to put color as a prop on SfIcon not only with our standard colors, or some custom color such as `#fff`, `rgb(255,255,255)`), and `lightgray` or nothing. What do you think ?

Reply:
on 2nd thoughts it is better to leave default colors as you said :) I restored styles for it, thanks!